examples_ch59x/debugprintfdemo/funconfig.h

```diff
@@ -0,0 +1,10 @@
+#ifndef _FUNCONFIG_H
+#define _FUNCONFIG_H
+
+#define FUNCONF_USE_HSI         0 // CH592 does not have HSI
+#define FUNCONF_USE_HSE         1
```

Comment:
How does the CH592 bootup?!?! Does it really boot right into the HSE?!?

Reply:
How can I check this?

Reply:
What happens if you don't touch clock settings, and boot the chip and short/remove he crystal?  Does the processor seize up?

Reply:
I'll have a try when I bring back alive my board

Reply:
actually what do you mean don't touch clock settings? not call `SystemInit()` at all? (that's basically all it does)
Would trying the usb bootloader with a shorted crystal also provide the info you need?

Reply:
I did everything I could here that would not break my board, but I was not able to get the processor to stop working whichever pins I shorted on the crystal. As not to block this PR on this, I'm going to resolve this as unresolved, sorry.